docs/self-managed/setup/guides/connect-to-an-oidc-provider.md

```diff
@@ -112,28 +108,27 @@ global:
 </TabItem>
 </Tabs>
 
-### Additional considerations
+<h3>Additional considerations</h3>
 
 For authentication, the Camunda components use the scopes `email`, `openid`, `offline_access`, and `profile`.
 
 </TabItem>
 <TabItem value="microsoftEntraId">
 
-### Steps
-
-1. Access the Entra ID admin area
-   and [register an application](https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-register-app).
-   After registering the app, the **Overview** page will contain a **Client ID**; make a note of this value as it will be
-   required later on.
+<h3>Steps</h3>
 
-2. Within the app registered in **Step 1**, [configure a platform](https://learn.microsoft.com/en-gb/entra/identity-platform/quickstart-register-app#configure-platform-settings)
-   of type `Web` for Operate, Tasklist, and Optimize. [Configure a platform](https://learn.microsoft.com/en-gb/entra/identity-platform/quickstart-register-app#configure-platform-settings)
-   of type `Single-page application` for Modeler. The expected redirect URI of the component you are configuring an app for can be found
-   in [component-specific configuration](#component-specific-configuration). Make sure the redirect URIs entered here match the redirect URI's configured in **Step 4**.
-
-3. Once you have registered a platform for your app a client secret needs to be created. To do this, see [adding a client secret](https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-register-app#add-a-client-secret). Make a note of the value of the client secret as it will be required later on.
+:::note
+Ensure you register a new application for each component.
+:::
 
-4. Set the following environment variables for the component you are configuring an app for:
+1. Within the Entra ID admin center, [register a new application](https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-register-app) for **each** component you would like to connect.
```

Comment:
Just wondering, if we have this in here `for **each** component you would like to connect` do we need the note above? Maybe though this is a case of making this point very clear :)